service.py

```diff
@@ -252,44 +310,7 @@ def playlistIndex(url, playlist):
 params = getParams()
 url = str(params['url'])
```

Comment:
If you are already on it, can improve the current code a bit and make sure result['url'] is string within the `getParams()` function?